docs/partners/plan-provisioning-woocommerce.md

```diff
@@ -0,0 +1,10 @@
+# Provisioning Woo Commerce Extensions
+If your Jetpack hosting partner account supports creating WooCommerce Extensions through the Jetpack hosting partner program this document provides additional information.
+
+All other documentation provided for the Jetpack partner hosting program is the same and valid for WooCommerce extension provisioning however there are a few extra details for provisioning extensions which are outlined below.
+
+## Provisioning Plan
```

Comment:
This section seems like a bit much. Perhaps we can summarize it and combine it with the above? Maybe something like:

> All other documentation provided for the Jetpack partner hosting program is the same and valid for WooCommerce extension provisioning, except the `plan` argument will need to change. At the moment, WooCommerce extension bundles are hosting partner specific, so please get in touch with us to get your plan slug if you do not already have it.